web/src/app/monitor/report/report.component.html

```diff
@@ -5,7 +5,7 @@
         <tr>
           <th>{{reportData.meta.rowDesc}}</th>
           <th class="align-center" *ngFor="let header of reportData.meta.header">
-            {{header}}</th>
+            {{getHeader(header)}}</th>
```

Comment:
We agreed that the backend defines the display values for the headers. ColumnHeader.displayName(). Please remove the client-side changes and modify the backend method.

Reply:
agree